### PR TITLE
Specify OpenGL (not OpenGLES) explicitly

### PR DESCRIPTION
**Pull request by teldosas**
Addresses #1177 

image

@simonsan @TheJJ That's what I get after this fix and all the errors go away.
It seems fixed... what do you think?

### CODE REVIEW

File: libopenage/gui/guisys/private/gui_application_impl.cpp [hunk 1]
```diff
@@ -17,6 +18,12 @@ std::shared_ptr<GuiApplicationImpl> GuiApplicationImpl::get() {
 
 	assert(!candidate || std::this_thread::get_id() == candidate->owner);
 
+	if (!candidate) {
+		QSurfaceFormat format;
+		format.setRenderableType(QSurfaceFormat::OpenGL);
```
[the remark this answers is not in the document]
Comment (simonsan) [reply libopenage/gui/guisys/private/gui_application_impl.cpp]: For us in the future: Could you please add a short comment/explanation on top of the if-statement why we had to do this and that it sets OpenGL as a kind of `standard value`. 👍

**Comment by teldosas** [comment]
@TheJJ what do you mean it's not in use? You mean that the `GuiApplicationImpl` is only instantiated once anyway?

**Comment by teldosas** [comment]
@TheJJ I'm not sure if it's written to but I think it's required because it instantiates the a `QGuiApplication` object which is necessary for qt. Not 100% sure though

**Comment by TheJJ** [comment]
kay :)

**Pull request merged**